core-bundle/src/Routing/RouteProvider.php

```diff
@@ -488,9 +491,15 @@ private function findPages(array $candidates): array
             $conditions[] = 'tl_page.alias IN ('.implode(',', $aliases).')';
         }
 
+        $options = [];
+
+        if (!empty($set)) {
+            $options['order'] = Database::getInstance()->findInSet('alias', $set);
```

Comment:
We should not use the old database connection in the new code

Reply:
I do not like this, either. We could use `FIND_IN_SET()` directly instead?

Reply:
Changed in 07c629278061b4576a55aff83745d7a446a696ae.